.github/scripts/createCommit.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+TARGET_TAG=$1
+
+git config user.email "broadbot@broadinstitute.org"
+git config user.name "broadbot"
+git_hash=$(git rev-parse --short "$GITHUB_SHA")
+
+# Code borrowed from https://github.com/fsaintjacques/semver-tool/blob/master/src/semver
```

Comment:
Helpful to have the reference link above. Would be even better to comment on whether the help text on the link applies to this script and if not, clarify the distinctions here as the details of what's going on don't look straightforward. May also be good to add a comment as to how this script is used.